data/types/alertable/alertable.go

```diff
@@ -0,0 +1,31 @@
+package alertable
+
+// Package alertable extracts types used when querying data used to evaluate
+// alerts configurations.
+//
+// I tried to put these in the data/types package (as AlertableParams and
```

Comment:
I don't think this package should be under `data/types`. All other types in the package are device data structs.

Reply:
> I don't think this package should be under `data/types`. All other types in the package are device data structs.

Where would you like to see it live?

Reply:
I'll move it to data/store... is that better? Just let me know.

Reply:
`data/store` is probably ok